README.md

```diff
@@ -141,6 +141,7 @@ Websites with lists of relays and their performance/health:
 - [Minds](https://www.minds.com/) - open source social network. Supports reading and creating posts using the Nostr protocol
 - [more-speech](https://github.com/unclebob/more-speech)![stars](https://img.shields.io/github/stars/unclebob/more-speech.svg?style=social) - desktop client for nostr written in Clojure
 - [nblog](https://github.com/jacany/nblog)![stars](https://img.shields.io/github/stars/jacany/nblog.svg?style=social) - a self-host nostr ghost blog
+- [ni.py](https://github.com/0n4t3/nipy) CLI post only client for Nostr and Activity Pub written in Python.
```

Comment:
Hey, sorry, updated the pull request with the proper formatting and also added a note on the AT Protocol since I got that working last night. (On mobile, hopefully didn't just break something else in the meantime) That alright? Was the last protocol I wanted to implement so hopefully no more changes would be needed. Thanks.

Reply:
My suggestion is just nitpicking. I think that's alright :)